release/downloadIstioCandidate.sh

```diff
@@ -34,10 +34,21 @@ fi
 NAME="istio-$ISTIO_VERSION"
 URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
 echo "Downloading $NAME from $URL ..."
-curl -L "$URL" | tar xz
-# TODO: change this so the version is in the tgz/directory name (users trying multiple versions)
-echo "Downloaded into $NAME:"
-ls "$NAME"
+curl -L "$URL" | tar xz 
+echo ""
+echo "Istio $ISTIO_VERSION Download Complete!"
+echo ""
+echo "Istio has been successfully downloaded into the $NAME subfolder on your system. 🎉\n"
+echo ""
 BINDIR="$(cd "$NAME/bin" && pwd)"
-echo "Add $BINDIR to your path; e.g copy paste in your shell and/or ~/.profile:"
-echo "export PATH=\"\$PATH:$BINDIR\""
+echo "Next Steps:\n"
+echo "To add Istio to a Kubernetes cluster, see https://istio.io/docs/setup/kubernetes/#evaluating-istio"
+echo ""
+echo "To configure the istioctl client tool for your workstation,"
+echo "add the $BINDIR directory to your environment path variable with:\n"
+echo "\t export PATH=\"\$PATH:$BINDIR\"\n"  
+echo ""
+echo "Then run:"
```

Comment:
We don't need "Then run:", because compatibility checking isn't part of the previous step.  How about "Check if your Kubernetes cluster is compatible with Istio by running" ?

Reply:
Yep- also makes sense.